MIGRATION.md

```diff
@@ -0,0 +1,65 @@
+## Migrating from zalando/jackson-datatype-money to FasterXML/jackson-datatypes-misc
+
+This document provides guidance on how to migrate from the `zalando/jackson-datatype-money` module to the new
+`FasterXML/jackson-datatypes-misc` module.
+
+# Changes in Dependency
+
+Replace the dependency in your `pom.xml` or build configuration from:
+
+```xml
+
+<dependency>
+    <groupId>org.zalando</groupId>
+    <artifactId>jackson-datatype-money</artifactId>
+    <version>1.3.0</version>
+</dependency>
+```
+
+to:
+
+```xml
+
+<dependency>
+    <groupId>com.fasterxml.jackson.datatype</groupId>
+    <artifactId>jackson-datatype-moneta</artifactId>
+    <version>2.19.0</version>
```

Comment:
What about leaving this free or putting `<version>pick-recent</version>` and add a hint to the text that the migration guide was validated for version `2.19.0`